server/internal/heavy/components.go

```diff
@@ -256,9 +256,11 @@ func newComponents(ctx context.Context, cfg configuration.Configuration, genesis
 		h.DropModifier = drops
 		h.PCS = CryptoScheme
 		h.PulseAccessor = Pulses
+		h.PulseCalculator = Pulses
 		h.JetModifier = jets
 		h.JetKeeper = jetKeeper
 		h.Sender = WmBus
+		h.PulseDelta = insolar.PulseNumber(cfg.Pulsar.NumberDelta)
```

Comment:
We can't use pulsar config inside a heavy node. Pulse is a completely standalone entity. You can see, that during working process of the network, Pulsar section of configs are empty for all nodes, except Pulsar